sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/screens/AppBar.kt

```diff
@@ -7,12 +7,14 @@ import androidx.compose.runtime.Composable
 import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.tooling.preview.Preview
 import co.nimblehq.sample.compose.R
+import co.nimblehq.sample.compose.ui.theme.AppTheme.colors
 import co.nimblehq.sample.compose.ui.theme.ComposeTheme
 
 @Composable
 fun AppBar(@StringRes title: Int) {
     TopAppBar(
-        title = { Text(text = stringResource(title)) }
+        title = { Text(text = stringResource(title)) },
```

Comment:
Title color is no longer white, as compared to your PoW 🙇 

Also, the progress-bar is showing purple, should we fix this? 

<img width="351" alt="Screen Shot 2023-01-17 at 16 01 48" src="https://user-images.githubusercontent.com/18277915/212854562-e1898ec1-3989-438a-8518-f94eb1db4443.png">

<img width="347" alt="Screen Shot 2023-01-17 at 16 06 40" src="https://user-images.githubusercontent.com/18277915/212855529-2cd114af-5e62-4ab5-96ff-ad7a8cd10635.png">

Reply:
@Tuubz, I feel it's out of scope because the color differs between the `dark/light` theme. I think it makes sense for now, So if we want to change the style, Shell we work on another issue instead? What do you think? cc @luongvo 

<img width="369" alt="Screen Shot 2566-01-19 at 11 01 31" src="https://user-images.githubusercontent.com/28002315/213352283-c2b3b992-ebdc-4fdf-91ff-fb8d7a8f54f4.png">

 
<img width="369" alt="Screen Shot 2566-01-19 at 11 01 19" src="https://user-images.githubusercontent.com/28002315/213352290-cb5b7483-b615-487a-808d-6d1eed3dab88.png">

Reply:
@Tuubz Sorry, I did not go too deep for this customization. I focused on how to structure the App theme & styles correctly. For this title bar text color:

- It will be in white with a Dark theme.
![Screenshot_20230119-113144](https://user-images.githubusercontent.com/16315358/213355610-e5786272-b94a-4cf2-ab2c-9de96e2ba9e2.png)

- It will be in black with a Light theme.
![Screenshot_20230119-113054](https://user-images.githubusercontent.com/16315358/213355621-4ee1a147-83ec-4db4-9a60-0ca3ea90639d.png)

Reply:
@luongvo I see, thank you for replying during your precious holidays. Let's keep it as it is then 🚀

Reply:
@Tuubz The POW has been updated.